package.json

```diff
@@ -14,10 +14,11 @@
   "dependencies": {
     "@babel/compat-data": "^7.8.6",
     "@bcrs-shared-components/certify": "^1.0.3",
+    "@bcrs-shared-components/contact-info": "1.0.3",
     "@bcrs-shared-components/detail-comment": "^1.0.4",
     "@bcrs-shared-components/enums": "^1.0.0",
     "@bcrs-shared-components/fee-summary": "^1.0.1",
-    "@bcrs-shared-components/interfaces": "^1.0.2",
+    "@bcrs-shared-components/interfaces": "^1.0.10",
```

Comment:
Interfaces is up to 11 now.

Reply:
kk, will update when i bump the Contact version.